src/LibraryInstaller.Vsix/Microsoft.Web.LibraryInstaller.Vsix.csproj

```diff
@@ -53,7 +53,7 @@
     <Compile Include="Commands\RestoreSolutionCommand.cs" />
     <Compile Include="Commands\RestoreCommand.cs" />
     <Compile Include="Commands\CleanCommand.cs" />
-    <Compile Include="Commands\InstallLibraryCommand.cs" />
+    <Compile Include="Commands\InstallLibraryCommand.cs" Condition="$(DefineConstants.Contains(UI_ENABLED))"/>
```

Comment:
Could you refactor this into a separate ItemGroup with all of the conditional files?  Like below with the Page items.